src/v/features/feature_table.h

```diff
@@ -455,8 +455,8 @@ inline constexpr std::array feature_schema{
     feature_spec::prepare_policy::always},
   feature_spec{
     release_version::v25_2_1,
-    "panda_linking_dr",
-    feature::panda_linking_dr,
+    "cluster_linking_dr",
+    feature::cluster_linking_dr,
```

Comment:
all good here, but something i just learned is that feature table uses the feature name here as the feature identifier, not the enum above (which i had incorrectly assumed) -- so renames like this could be problematic if done across versions.